src/defaultPlugins.ts

```diff
@@ -105,4 +114,25 @@ export const defaultPlugins = {
             return numbersWithoutTPAParams[0];
         }
     },
+    smartContrast: (baseColor, varColor) => {
```

Comment:
Does the `baseColor` is the text color and the `varColor` is the current bgColor?

For me, it is not clear from the function name `smartContrast` what does this function exactly do. Maybe we should rename it for something like `getBgColorWithSmartContrast`? WDYT?

Reply:
yeah, I wasn't sure about how to name the variables.
So it doesn't have to be text color and background, it can be any two colors.

The thing is, one color is the base color, which is constant, the other color is the color that can change.
So maybe it should be something like:
```js
smartContrast: (color, suggestionColor) => {
```

or something like that

Reply:
yeah, I think that (color, suggestionColor) is better :)